src/DataReader/xml_parser.rs

```diff
@@ -140,6 +140,7 @@ fn collect_edges(xml_edges: FindChildren) -> Vec<Edge> {
             }
         }
         let edge = component::Edge {
+            id: "NotImplemented".to_string(), //NEEDS TO BE FIXED
```

Comment:
Maybe fix this?

Reply:
Skal ikke fixes, da vi ikke skal understøtte xml til GUI (noget sebastian sagde)